.pre-commit-config.yaml

```diff
@@ -0,0 +1,15 @@
+repos:
+  - repo: https://github.com/norwoodj/helm-docs
```

Comment:
@ernesgonzalez33 @arturohu we need still decide if we want this on the CI or as pre-hook, but nice work!

Reply:
Should we remove it for the moment?

Reply:
Lest discuss it all together.

Reply:
I would leave it as a first step and build the ci based on that work in the second step.